src/Propagation/ServerTiming/composer.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "open-telemetry/opentelemetry-propagation-server-timing",
```

Comment:
let's call it `open-telemetry/contrib-propagator-server-timing` (which is similar to existing propagator names, and assumes that eventually we'll move it to core and replace with `extension-propagator-server-timing`